File: benchmarks/fixtures/go/noisy.go
package main
import ("fmt"; "os")
func main(){ for i:=0;i<4000;i++ { fmt.Printf("go fixture line %d deterministic output\n",i) }; fail:=len(os.Args)>1&&os.Args[1]=="--fail"; if fail {fmt.Fprintln(os.Stderr,"go fixture intentional failure");os.Exit(7)};fmt.Fprintln(os.Stderr,"go fixture success") }

File: benchmarks/fixtures/rust/noisy.rs
use std::env;
fn main() { for i in 0..4000 { println!("rust fixture line {} deterministic output", i); } let fail=env::args().any(|a| a=="--fail"); eprintln!("rust fixture {}", if fail {"intentional failure"} else {"success"}); if fail {std::process::exit(7)} }
